src/client/index.ts

```diff
@@ -20,9 +20,11 @@ import {
   decodeXAddress,
 } from "ripple-address-codec";
 
+import { bind } from "..";
```

Comment:
why call this bind? the whole point i thought was to call it `partial`

Reply:
Ah you're right, slipped my mind!